root_data.tf

```diff
@@ -29,3 +29,19 @@ data "aws_ssm_parameter" "workflow_pat" {
 data "aws_ssm_parameter" "slack_webhook_url" {
   name = "/mgmt/release/slack/webhook"
 }
+
+data "aws_ssm_parameter" "gpg_passphrase" {
+  name = "	/mgmt/github/gpg/passphrase"
```

Comment:
Is there a reason for the whitespace at the beginning of the names of the params?

Reply:
I've removed this too.